jekyll/_cci2/configuration-cookbook.md

```diff
@@ -65,89 +60,68 @@ Configuration Recipe | Description
 [Using Amazon Elastic Container Service for Kubernetes (Amazon EKS)](#using-amazon-elastic-container-service-for-kubernetes-amazon-eks) | This section describes how you can use the Amazon ECS service for Kubernetes for Kubernetes-related tasks and operations.
 [Enabling Custom Slack Notifications in CircleCI Jobs](#enabling-custom-slack-notifications-in-circleci-jobs) | This section describes how you can enable customized Slack notifications in CircleCI jobs.
 [Using Logic in Configuration](#using-logic-in-configuration) | This section describes how you can use pipeline values & parameters to select the work to perform.
+{: class="table table-striped"}
 
-## Deploying Software Changes to Amazon ECS
+## Deploy changes to Amazon ECS
```

Comment:
Changing `#deploying-software-changes-to-amazon-ecs` on line 58 to `#deploy-changes-to-amazon-ecs` should hopefully fix the failing build.

Reply:
Ahhhh thank you!!